shapeshift/web
# feat: adding japanese translations (ja) and language option

## Description

- Adds the Japanese language in the UI
- Adds Japanese translations (ja)

Before merging, could someone in the Engineering WS please deploy this branch on a testing instance so our translator/reviewer can check the translations in-app. It might take few days to complete the review (based on their schedule). Once this will be done I will convert this from a Draft to a PR.

Also requesting @twblack88 's approval from Product's side to add the language in the app, there have been discussions about this on Discord, but I'm not sure it has ever been formally approved/added to the roadmap, so please can you confirm it here?



## Pull Request Type


## Risk
Low. New option in the UI. Any missing translation will fallback to English.



## Testing

* Check that Japanese (日本語) is available and working in the Language Selector (on the welcome screen) and in the Settings modal
* Check if the UI is broken anywhere in the app due to the new strings.
* Check that the Japanese language is detected if it's the default language in the browser during the first visit (you can use the Clear Cache feature for this as it clears this setting too).



### Engineering
☝️ 


### Operations
☝️ 


## Screenshots (if applicable)
Few mobile and PC screenshots:
![image](https://github.com/shapeshift/web/assets/88804546/c7f659db-0d39-4996-a815-4e3b0c9d6caa)

![image](https://github.com/shapeshift/web/assets/88804546/c10a6c78-107e-4bf5-a460-66661c063dad)

![image](https://github.com/shapeshift/web/assets/88804546/a0586af9-147c-4a80-ba91-7863743c8d3f)

![image](https://github.com/shapeshift/web/assets/88804546/188515d6-8fa0-4ec7-a268-3b68952c8f1c)

![image](https://github.com/shapeshift/web/assets/88804546/7e9ef4db-f199-4c70-8074-0af04e3efd30)

![image](https://github.com/shapeshift/web/assets/88804546/331c4a89-6ad2-4daf-82f5-5787e05f753d)
